src/auto/builder_cscope.rs

```diff
@@ -0,0 +1,60 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+use BuilderScope;
+
+glib_wrapper! {
+    pub struct BuilderCScope(Object<gtk_sys::GtkBuilderCScope, gtk_sys::GtkBuilderCScopeClass, BuilderCScopeClass>) @implements BuilderScope;
```

Comment:
This should probably not be bound. It's for resolving via C symbols, that doesn't seem very useful in Rust and... unsafe

Reply:
It's not bound neither, I have ended up disabling it later